app/src/main/java/app/grapheneos/camera/ui/activities/MainActivity.kt

```diff
@@ -191,10 +191,17 @@ open class MainActivity : AppCompatActivity(),
             previewView.width.toFloat(), previewView.height.toFloat()
         )
 
-        val autoFocusPoint = factory.createPoint(
-            previewView.width / 2.0f,
-            previewView.height / 2.0f, qrOverlay.size
-        )
+        val autoFocusPoint = if (shouldUseFocusRegion()) {
```

Comment:
Is there a some cleaner Kotlin way to do this?

Reply:
I don't think there's a better way to do this part.